src/server.js

```diff
@@ -40,11 +43,24 @@ function calculateReward(value, volume) {
     return "Doesn't even fit in an Orca!";
   };
 
+  // If someone pastes in units of millions of ISK, convert to units of ones of ISK.
+  // 20.25 -> 20,250,000
+  if (value < 100000) {
```

Comment:
This seems super confusing to me.  So we just don't allow low-value contracts at all with this tool?  This needs documentation for a "minimum value contract" below which your input is treated as millions.